app/WeightConvertor.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace ConvertorApp;
+class WeightConvertor {
+	/*
+	 * This function Converts the Input Weight from Gram To Kilogram
+	 */
+	public function convertGramToKilogram($weight) {
+		if (is_string($weight)) {
```

Comment:
All your validations are same. You can create a base class and add validate function to that class and inherit that class in each `Convertor` class. Will save lot of code.